zerver/tests/test_home.py

```diff
@@ -221,11 +221,11 @@ def _sanity_check(self, result):
 
     def test_terms_of_service(self):
         # type: () -> None
-        email = 'hamlet@zulip.com'
+        user = self.example_user('hamlet')
+        email = user.email
         self.login(email)
 
         for user_tos_version in [None, '1.1', '2.0.3.4']:
-            user = get_user_profile_by_email(email)
             user.tos_version = user_tos_version
             user.save()
```

Comment:
I read all the changes looking for this sort of thing, so I'm pretty sure no others got through.